src/components/FAQ/FAQ.scss

```diff
@@ -51,7 +51,7 @@
     margin-bottom: 0px;
     font-weight: 400;
     font-size: 16px;
-    font-family: 'Clan Offc Pro Book';
+    font-family: 'Clan Offc Pro Book', -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,"Noto Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
```

Comment:
Can you please explain this diff @IntegralTeam specifically what `-apple-system` does here

Reply:
Every font, loading we it or not, should has fallback font-faces. That's the rule of good markup.